### DISCUSSION TITLE
Inverted right thumbstick

### DISCUSSION THREAD

**Discussion by ROGsamurai** [ndeadly/MissionControl#397] [General]
anyway to make the right stick when i press right it goes left and when i press left it goes right?

**Comment by ndeadly** [comment]
Not currently. There are a bunch of controller settings in the works for a future release, along with a Tesla overlay to control them. This is one of the configuration options already. If you're interested you can check out the [pull request](https://github.com/ndeadly/MissionControl/pull/302) from @Banz99 to see the current progress. No ETA on release though

**Reply by ROGsamurai** [reply]
the reason i ask this is cause i have a mod out right now and the control are basicly screwed. This is what the controls are now and this is for the super mario 3d all-stars game n64 emulator

L is R
R is R
Z is Z
Start is Start
A is A
B is A

Dpad Is The Same As Anglog Stick

C-buttons

C-Left is C-Right
C-Right is C-Left
C-Down is C-Down
C-Up Is C-Up

will this mod be able to fix this and will a donation help this feature any faster cause i want this to be the best trilogy pack out for this game check it out if you want

https://gbatemp.net/threads/the-legend-of-zelda-35th-anniversary-sm3das-rom-hack-w-i-p.603337/

**Reply by ndeadly** [reply]
I think you would be better off trying to get to the bottom of why the controls are messed up in the first place. Presumably some things already needed to be patched to make this mod work? Maybe it's just a matter of patching the control handling code.

I'm sure Mission Control could in theory fix your problem, but it sounds like more of a band-aid solution. For starters, it will only allow the controls to be modified for bluetooth connections. Handheld mode and USB controllers would still have the messed up controls. You would also need to toggle the changes on/off manually when switching between your mod and anything else as there is no mechanism for applying controller profiles on a per-title basis.

Donations could certainly help me consider prioritising development of certain features, but it is mostly an issue of free time/motivation at the moment.

**Reply by ROGsamurai** [reply]
yeah im not good a coding just editing files. that would probably be in the main file that in the exefs folder. I had someone patch the save file location for me cause the saves were getting overwritten.

If your willing to look into the controls I'll gladly pay you something for it np

**Reply by ndeadly** [reply]
Can take a look later tonight but can't make any promises. Are all games in the trilogy affected in the same manner or does it differ between them?

**Reply by ROGsamurai** [reply]
all the same, they use the same main file kinda. just if you notice anything about controls i can let the person that patched the save location know what you found if anything

it's just a layeredfs mod
